src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -81,8 +83,9 @@ def _main(settings, optlist, args):
         else:
             log.error("Error: No exception thrown, but XUnit results not created")
             test_executor.report_error(settings, failure_type="XUnitTestFailure")
+            return WORKITEM_FAILED
```

Comment:
It might be useful to return the xunit exit code if it's not zero here, in case it has some semantic meaning.